pkg/ccl/backupccl/testdata/backup-restore/descriptor-conflicts

```diff
@@ -3,8 +3,10 @@
 # appropriately during a cluster restore. The conflicting system tables in the
 # restoring cluster should be copied to a descriptor ID higher than any
 # descriptor in the backup.
+#
+# disabled for tenants because the create-dummy-system-table cmd cannot be run within a tenant
```

Comment:
Initially, because of the codec, but i attempted to refactor the dd command to be MT friendly, but then ran into another snag. the following is not available to tenants
```
ALTER TABLE foo.bar.bat CONFIGURE ZONE USING gc.ttlseconds=999;
```

https://github.com/cockroachdb/cockroach/issues/49854?version=v22.2